pandas/core/arrays/base.py

```diff
@@ -350,6 +351,38 @@ def __iter__(self):
         for i in range(len(self)):
             yield self[i]
 
+    def __eq__(self, other: ABCExtensionArray) -> bool:
```

Comment:
These will need to be added to the list of abstract methods in the "Notes" section of the class docstring.

Reply:
Done